### PR TITLE
Feat/dark mode

### PR DESCRIPTION
**Pull request by emmanuelgaravito**
Added Dark Mode functionality.

### CODE REVIEW

File: app/[locale]/(auth)/layout.tsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
 // @Components
 import LanguageSelector from "@/components/shared/LanguageSelector";
+import { ModeToggle } from "@/components/theme-toggle";
 
 type AuthLayoutProps = { children: React.ReactNode };
 
 export default function AuthLayout({ children }: AuthLayoutProps) {
 	return (
-		<main className="bg-white md:bg-slate-50">
+		<main className=" ">
```
Comment (ingcarlostello) [inline app/[locale]/(auth)/layout.tsx]: if there are no classes remove className

File: app/[locale]/dashboard/layout.tsx [hunk 2]
```diff
@@ -14,12 +15,15 @@ const Layout = ({ children }: DashboardLayoutProps) => {
...
+			<div className="flex justify-end  gap-2 pr-4 pt-2">
+				<div className="hidden sm:block">
+					<ModeToggle />
+				</div>
 				<LanguageSelector />
 			</div>
 
 			<div className="flex">
-				<div className="bg-slate-50 w-2/12 border-r h-screen max-[767px]:hidden md:block md:w-1/4 lg:w-1/6">
+				<div className="  w-2/12 border-r h-screen max-[767px]:hidden md:block md:w-1/4 lg:w-1/6">
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/layout.tsx]: delete space

File: app/[locale]/dashboard/profile/page.tsx [hunk 3]
```diff
@@ -30,16 +30,14 @@ const Profile = () => {
 			{" "}
 			<Form {...form}>
 				<div className="flex justify-center items-center">
-					<div className="bg-white md:w-3/6 lg:w-3/6 xl:w-2/4 p-8 flex flex-col gap-10">
+					<div className=" md:w-3/6 lg:w-3/6 xl:w-2/4 p-8 flex flex-col gap-10">
```
Comment (ingcarlostello) [inline app/[locale]/dashboard/profile/page.tsx]: delete space

**Review by ingcarlostello — CHANGES_REQUESTED** [review]

File: app/[locale]/(auth)/layout.tsx (hunk 1 above)
Comment (emmanuelgaravito) [reply app/[locale]/(auth)/layout.tsx]: Done

File: app/[locale]/dashboard/layout.tsx (hunk 2 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/layout.tsx]: Done

File: app/[locale]/dashboard/profile/page.tsx (hunk 3 above)
Comment (emmanuelgaravito) [reply app/[locale]/dashboard/profile/page.tsx]: Done

**Review by ingcarlostello — APPROVED** [review]

**Pull request merged**